src/components/auto-complete/auto-complete.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { OptionList } from '../selection-list/selection-list';
+import { SBComponent } from 'stylable-react-component/dist/stylable-react';
+import { root } from 'wix-react-tools';
+import { noop } from '../../utils/noop';
+import style from './auto-complete.st.css';
+
+export interface AutoCompleteProps extends OptionList {
+    value?: string;
+    onChange?: (value: string) => void;
```

Comment:
`onChange` should have a more complete event interface like: `(e: React.ChangeEvent<HTMLInputElement>)) => void` or an interface that extends that of react and adds additional fields. Look at Image component for pattern